pages/school-administrators.md

```diff
@@ -93,17 +93,25 @@ relatedlinks:
+<va-featured-content uswds>
+  <h3 slot="headline">Upload documents to VA</h3>
+  <p>
+    The Education File Upload Portal is for certifying officials to upload documents supporting compliance actions, designating or removing certifying officials, and more. The full list of accepted documents is found in the upload portal.
```

Comment:
I noticed this wording is a little different than what is in the screenshot for #211. Just want to make sure that's all good from a Product/Design perspective?

Reply:
Just checked with design. The most up-to-date wording is in the Figma file, rather than the screen shot, and I followed the Figma.